src/SkynetServer.Shared/Migrations/20190102220945_ModelImplementation.cs

```diff
@@ -0,0 +1,334 @@
+﻿using System;
```

Comment:
Are these files generated by VS / EF? If so, we could ignore them

Reply:
Migrations are generated by a PowerShell script in order to update the database model while keeping existing data. As they are generated manually and are manually edited sometimes, they have to be checked in.

Reply:
Alright.